### ISSUE TITLE
Can't get positioning and everything to show when popover within scrolling div

### ISSUE THREAD

**Issue by asong91** [twbs/bootstrap#16442]
So here's a JSFiddle of the problem I ran into.  http://jsfiddle.net/Br4Zg/1115/

The popover is getting cut off at the top. If I add "container:body" to the popover properties I lose the functionality of the popover being relative to the element in the table when I scroll.

**Comment by twbs-lmvtfy** [comment]
Hi @asong91!

You appear to have posted a live example (http://fiddle.jshell.net/Br4Zg/1115/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 57, column 3 thru line 57, column 8: Start tag ```` body ```` seen but an element of the same type was already open.
* line 141, column 1 thru line 141, column 6: Start tag ```` body ```` seen but an element of the same type was already open.
* line 143, column 1 thru line 143, column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 60, column 3 thru line 60, column 35: Unclosed element ```` div ````.
* line 59, column 1 thru line 59, column 83: Unclosed element ```` div ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Closing per https://github.com/twbs/bootstrap/issues/15367#issuecomment-99209644 as CantFix/WontFix for Bootstrap v3. Bootstrap v4 will hopefully be better.
